serviceRequests/requestType.py

```diff
@@ -1,5 +1,6 @@
 from hesburgh import heslog
 import urllib2
+import socket
 
 class RequestType(object):
```

Comment:
I'd recommend adding timeout and max retry as class variables and then reference them later instead of magic numbers. This also reduces the number of places where we'd have make updates in the future if needed. Code snippets not tested!
```suggestion
class RequestType(object):
    MAX_RETRIES = 2
    URL_TIMEOUT = 8
```

then later on...
```python
response = urllib2.urlopen(req, timeout=self.URL_TIMEOUT)
```
and
```python
if retryCount < self.MAX_RETRIES:
```

Reply:
Fair enough. :)